### ISSUE TITLE
MIDS Element - CollectionDate

### ISSUE THREAD

**Issue by RBGE-Herbarium** [tdwg/mids#24]
**Summary**

MIDS information element: 

DwC term (latest, 2014-11-08): 

ABCD term name(3.0): 

Recommendations:

**Comment by hardistyar** [comment]
Should we adopt the DwC term name 'eventDate' instead of using CollectionDate?

**Comment by matdillen** [comment]
@hardistyar I would not recommend `CollectionDate`, in particular given the existence of the unrelated `CollectionCode`. We could reuse the terminology of `Event` or `Gathering`. Another alternative could be `dateCollected`.

If we move this to MIDS 1, is `1800-01-01/2020-01-26` a valid value?

**Comment by hardistyar** [comment]
@matdillen I agree it is best to use already established terms such as eventDate. The problem we have is to know whether we should preferably base on Darwin Core terms or ABCD terms, or an aligned term. See #41.

A range would be a valid value.

**Comment by emhaston** [comment]
Element information updated based on discussion at meeting 18, 13 October

CollectingDate
Required: Yes (Biological) No (Geo & Palaeo)
How do we handle verbatim dates?
We should definitely make a recommendation, perhaps in line with DwC/ABCD date fields?
Could be the same recommendation for all dates in MIDS.

**Comment by emhaston** [comment]
Name updated from CollectionDate to dateCollected as a proposal based on Mathias's suggestion.

**Comment by elspethhaston** [comment]
Accepted in specification for public review.
